_pages/about-us/teams/oa

```diff
@@ -47,4 +47,4 @@ TTS OA aims to redefine the culture of government acquisition of information tec
 
 #### Still have questions?
 
-Ask in Slack: [#acquisition](https://gsa-tts.slack.com/messages/acquisition)
+Ask in Slack: [#tts-oa-internalbuy](https://gsa-tts.slack.com/messages/acquisition)
```

Comment:
Does the link here need to change to https://gsa-tts.slack.com/messages/CD4F3TGB1/ ?

Reply:
Yes!